### ISSUE TITLE
TypeError ufunc`true_divide` - networkx DiGraph + netgraph 

### ISSUE THREAD

**Issue by HamedMP** [paulbrodersen/netgraph#9]
Getting the following error when drawing a networkx Digraph(). 

networkx version: 2.0
netgraph version: 3.1.2
python 3

```shell
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-63-2bd83e0b2753> in <module>()
----> 1 netgraph.draw(G, draw_arrows=True)

/anaconda3/lib/python3.6/site-packages/netgraph/_main.py in draw(graph, node_positions, node_labels, edge_labels, edge_cmap, ax, **kwargs)
     77         # but if the variance in weights is large, this typically results in less
     78         # visually pleasing results.
---> 79         edge_color  = get_color(edge_weight, cmap=edge_cmap)
     80         kwargs.setdefault('edge_color',  edge_color)
     81 

/anaconda3/lib/python3.6/site-packages/netgraph/_main.py in get_color(mydict, cmap, vmin, vmax)
    313     #  - the colormap midpoint is at zero-value, and
    314     #  - negative and positive values have comparable intensity values
--> 315     values /= np.nanmax([np.nanmax(np.abs(values)), abs(vmax), abs(vmin)]) # [-1, 1]
    316     values += 1. # [0, 2]
    317     values /= 2. # [0, 1]

TypeError: ufunc 'true_divide' output (typecode 'd') could not be coerced to provided output parameter (typecode 'l') according to the casting rule ''same_kind''
```

**Comment by paulbrodersen** [comment]
Bloody python 3 and their save casting. Thanks for the heads-up.

**Comment by paulbrodersen** [comment]
Fixed by 39b56c0ff4d01525629e7c6273a18d6441539446.
